tests/test/system/test_network.cpp

```diff
@@ -42,4 +42,31 @@ TEST_CASE("Test basic network properties", "[faaslet]")
     faabric::util::setEnvVar("NETNS_MODE", original);
     conf.reset();
 }
+
+TEST_CASE("Test running out of namespaces", "[faaslet][network]")
+{
+    conf::FaasmConfig& conf = conf::getFaasmConfig();
+    std::string originalNetNsMode = conf.netNsMode;
+
+    faabric::util::setEnvVar("NETNS_MODE", "on");
+    conf.reset();
```

Comment:
The easiest way to do this sort of config change is:

```
TEST_CASE_METHOD(FaasmConfTestFixture, "Test running out of namespaces", "[faaslet][network]") {
    conf.netNsMode = "on";

    // Do some testing
}
```

Then you don't need to call `getFaasmConfig()`, or handle environment variables or keeping track of the original value.